msgType.h

```diff
@@ -0,0 +1,133 @@
+#pragma once
+typedef enum msgType	//type of messages used as enum
+{
+	joinRequest=0x01,	//used in pairing mode for join request
+	joinAccept,			//used in pairing mode for join accept
+	joinConfirm,		//used in pairing mode for join confirmation
+	configReq,			//used in configuration mode for requesting configuration message
+	configMsg,			//used in configuration mode for sending configuration message
+	configAck,			//used in configuration mode for acknowledging configuration successfull
+	sensorData,			//used in monitoring mode for sending sensor data and battery level
+	ack,				//used for acknowledging a message received
+	motorStatus,		//used for acknowleging status of motor
+	alarm,				//used to send warning messages/error messages
+	initController		//used to notify level monitor that the controller is recently initiated
+}msgType; 
+
+typedef enum devType {
+	monitor = 0x01,
+	controller,
+	relayUnit
+}devType;
+
+typedef enum response {
```

Comment:
Is it showing result/error, then better use errorno
Status as 0 and others as numbers

Reply:
it is responses for acknowledging join requests, acknowledgements of messages etc.
Error states are dealed here. shall make the necessary changes

Reply:
it is responses for acknowledging join requests, acknowledgements of messages etc.
Error states are dealt with at this position. shall make the necessary changes.